### PR TITLE
Added option to choose if a link should open in new tab

### LINKED ISSUE #1081: Update edit link options
Ref: https://github.com/bigbinary/neeto-kb-web/issues/6085

- Add option to enable user to choose if a link opens in new tab
- Update labels

### PR DESCRIPTION
**Pull request by gaagul**
- Fixes #1081 

**Description**

- Added: option to choose if a link should open in new tab

**Checklist**

~~- [ ] I have made corresponding changes to the documentation.~~
~~- [ ] I have updated the types definition of modified exports.~~
- [x] I have verified the functionality in some of the neeto web-apps.
- [x] I have added the necessary label (patch/minor/major - If package publish
      is required).

**Reviewers**

@AbhayVAshokan _a

### CODE REVIEW

**Comment by gaagul** [comment]
@AbhayVAshokan Please let me know if the are any updates needed to the changes.

File: src/components/Editor/LinkPopOver.jsx [hunk 1]
```diff
@@ -154,6 +152,13 @@ const LinkPopOver = ({ editor }) => {
             style={{ width: "250px" }}
             onKeyDown={handleKeyDown}
           />
+          <Switch
+            checked={values.openInNewTab}
+            label={t("neetoEditor.common.openInNewTab")}
+            onChange={() => {
+              setFieldValue("openInNewTab", !values.openInNewTab);
+            }}
+          />
```
Comment (AbhayVAshokan) [inline src/components/Editor/LinkPopOver.jsx:161]: Use a `Checkbox` instead of a `Switch`.

**Review by AbhayVAshokan — CHANGES_REQUESTED** [review]
@gaagul _a: https://abhay-ashokan.neetorecord.com/watch/1594d856-27f7-4588-b480-ad74907ce98a

**Comment by gaagul** [comment]
@AbhayVAshokan _a Please have a look at the updates.

**Review by AbhayVAshokan — APPROVED** [review]

**Pull request merged**